### PR TITLE
Configurable json inclusions in Avro to Json serializing

### PR DESCRIPTION
**Pull request by magnusgundersen**

### CODE REVIEW

File: src/main/java/org/akhq/utils/AvroToJsonSerializer.java [hunk 1]
```diff
@@ -1,32 +1,49 @@
...
+    }
 
-        return MAPPER.writeValueAsString(map);
+    private ObjectMapper createObjectMapper(List<Include> jsonInclusions) {
+        ObjectMapper objectMapper = new ObjectMapper()
+            .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
+            .registerModule(new JavaTimeModule())
+            .registerModule(new Jdk8Module())
+            .setTimeZone(TimeZone.getDefault());
+        System.out.println("Inclusions: " + jsonInclusions);
```
Comment (tchiotludo) [inline src/main/java/org/akhq/utils/AvroToJsonSerializer.java:34]: remove this ;)

File: src/main/java/org/akhq/utils/AvroToJsonSerializer.java [hunk 2]
```diff
@@ -1,32 +1,49 @@
...
-        .setSerializationInclusion(JsonInclude.Include.NON_NULL)
-        .setSerializationInclusion(JsonInclude.Include.NON_EMPTY)
-        .registerModule(new JavaTimeModule())
-        .registerModule(new Jdk8Module())
-        .setTimeZone(TimeZone.getDefault());
+    private final ObjectMapper mapper;
 
-    public static String toJson(GenericRecord record) throws IOException {
-        Map<String, Object> map = AvroDeserializer.recordDeserializer(record);
+    public AvroToJsonSerializer(@Value("${akhq.avro-serializer.json.serialization.inclusions}") @Nullable List<Include> jsonInclusions) {
```
Comment (tchiotludo) [inline src/main/java/org/akhq/utils/AvroToJsonSerializer.java:23]: You are changing the default configuration, please add old default one on the application.yml please

**Review by tchiotludo — CHANGES_REQUESTED** [review]
Mostly good. 
Please just add : 
- configuration on application.example.yml 
- some doc on README.md about this configuration 
- minor review :arrow_up:

File: src/main/java/org/akhq/utils/AvroToJsonSerializer.java (hunk 2 above)
Comment (magnusgundersen) [reply src/main/java/org/akhq/utils/AvroToJsonSerializer.java:23]: Added the old default to src/main/resources/application.yml

```yaml
  avro-serializer:
    json.serialization.inclusions:
      - NON_NULL
      - NON_EMPTY
 ```

File: src/main/java/org/akhq/utils/AvroToJsonSerializer.java (hunk 1 above)
Comment (magnusgundersen) [reply src/main/java/org/akhq/utils/AvroToJsonSerializer.java]: Whoops.. done!

**Comment by tchiotludo** [comment]
Perfect, thanks a lot :+1:

**Pull request merged**